cmd/zdapd/zdapd.go

```diff
@@ -77,12 +77,18 @@ func main() {
 					{
 						Name:  "snap",
 						Usage: "creates a snap of a resource",
+						Flags: []cli.Flag{
+							&cli.BoolFlag{
+								Name:  "use-existing",
+								Usage: "only create snap if base exists",
+							},
+						},
```

Comment:
@J-Zeitler do you remember why we added this?

Reply:
I remember I had some trouble getting things setup with a base + snap. I think I did something where I manually created the base (expensive/takes time to do db replication) and then wanted to guarantee that it wasn't re-created when creating snaps. This may not be an issue when we have a scheduled workflow as long as it works. But the `use-existing` was a good safeguard when troubleshooting

Reply:
Yes that vaguely rings a bell, I guess we were replicating the db with a fast backup manually or something like that